crates/air_r_formatter/src/context.rs

```diff
@@ -8,16 +8,16 @@ use biome_formatter::BracketSpacing;
 use biome_formatter::CstFormatContext;
 use biome_formatter::FormatContext;
 use biome_formatter::FormatOptions;
-use biome_formatter::IndentStyle;
-use biome_formatter::IndentWidth;
-use biome_formatter::LineEnding;
-use biome_formatter::LineWidth;
 use biome_formatter::TransformSourceMap;
+use settings::IndentStyle;
+use settings::IndentWidth;
+use settings::LineEnding;
+use settings::LineWidth;
+use settings::MagicLineBreak;
```

Comment:
It's implicit here, but note that our `RFormatOptions` struct now uses our settings types